src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportOrchestratorJob.cs

```diff
@@ -109,34 +105,11 @@ public async Task<string> ExecuteAsync(JobInfo jobInfo, IProgress<string> progre
             {
                 cancellationToken.ThrowIfCancellationRequested();
 
-                if (currentResult.Progress == ImportOrchestratorJobProgress.Initialized)
-                {
-                    await ValidateResourcesAsync(inputData, cancellationToken);
-
-                    currentResult.Progress = ImportOrchestratorJobProgress.InputResourcesValidated;
-                    progress.Report(JsonConvert.SerializeObject(currentResult));
-
-                    _logger.LogJobInformation(jobInfo, "Input Resources Validated.");
-                }
-
-                if (currentResult.Progress == ImportOrchestratorJobProgress.InputResourcesValidated)
-                {
-                    await _importOrchestratorJobDataStoreOperation.PreprocessAsync(cancellationToken);
```

Comment:
Are we no longer disabling indexing for initial load $import??

Reply:
Yes. We are not using it in PROD for multiple reasons, so I remove it.

Reply:
This makes the future `$import` orchestrator changes much easier.